differ/differ.go

```diff
@@ -351,14 +357,40 @@ func processReportsByCluster(ruleContent types.RulesMap, storage Storage, cluste
 					Int("impact", impact).
 					Int(totalRiskAttribute, totalRisk).
 					Msg("Report with high impact detected")
-				if !shouldNotify(cluster, r) {
-					continue
+
+					// Notification backend
+				if shouldNotify(cluster, r, types.NotificationBackendTarget) {
```

Comment:
does it take into account, that one or second event target will be disabled_

Reply:
I think that should be managed in the [notifier.ProduceMessage(notificationMsg)](https://github.com/RedHatInsights/ccx-notification-service/blob/408fa1182b71ed3cdfc69fd9f8bc02ebb40b8ab1/differ/differ.go#L371) (now it is)

Reply:
WDYT?